src/app/store/ledger/bitcoin-key.slice.ts

```diff
@@ -0,0 +1,34 @@
+import { PayloadAction, createEntityAdapter, createSlice } from '@reduxjs/toolkit';
+
+import { BitcoinLedgerAccountDetails } from '@app/features/ledger/utils/bitcoin-ledger-utils';
+
+import { RootState } from '..';
+import { defaultKeyId } from '../keys/key.slice';
+
+interface PersistedBitcoinKeys extends BitcoinLedgerAccountDetails {
+  walletId: string;
+}
+
+const bitcoinKeyAdapter = createEntityAdapter<PersistedBitcoinKeys>();
+
+export const bitcoinKeySlice = createSlice({
```

Comment:
Why do I feel like it would be more appropriate to have all the naming here be plural?